app/models/shortener/shortened_url.rb

```diff
@@ -25,33 +25,46 @@ def self.clean_url(url)
   # generate a shortened link from a url
   # link to a user if one specified
   # throw an exception if anything goes wrong
-  def self.generate!(destination_url, owner: nil, custom_key: nil, expires_at: nil, fresh: false)
+  def self.generate!(destination_url, owner: nil, custom_key: nil,
+                     expires_at: nil, fresh: false, meta: nil)
```

Comment:
Since it is already part of the gem, we can open a new ticket to change the name.